### PR TITLE
Implement volume control & update dependencies

### LINKED ISSUE #5: Volume control
Hello, is there any way to control the volume?

### PR DESCRIPTION
**Pull request by vmasto**
Implements volume control and updates dependencies (and React to the most recent version).

Resolves #5

### CODE REVIEW

File: examples/src/Example.js [hunk 1]
```diff
@@ -29,13 +32,16 @@ export default class Example extends React.Component {
...
+        onVolumeUp={() => this.setState({volume: volume >= 100 ? volume : volume+10})}
+        onVolumeDown={() => this.setState({volume: volume <= 0 ? volume : volume-10})}
         duration={this.state.currentSong ? this.state.currentSong.duration : 0}
         position={this.state.position} />
       {this.state.currentSong &&
         <Sound
           url={this.state.currentSong.url}
           playStatus={this.state.playStatus}
           playFromPosition={this.state.position}
+          volume={this.state.volume}
```
Comment (leoasis) [inline examples/src/Example.js]: You can use `volume` from above

File: examples/src/index.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { render } from 'react-dom';
 import Example from './Example';
 
-React.render(<Example />, document.getElementById('app'));
+render(<Example />, document.getElementById('app'));
```
Comment (leoasis) [inline examples/src/index.js]: Use the `ReactDOM` namespace instead of just `render`

File: package.json [hunk 3]
```diff
@@ -19,21 +19,22 @@
   "author": "Leonardo Andres Garcia Crespo <leoasis@gmail.com>",
   "license": "ISC",
   "homepage": "https://github.com/leoasis/react-sound",
-  "peerDependencies": {
-    "soundmanager2": "^2.97.20150601-a",
-    "react": ">=0.13.3 <16.0.0"
+  "dependencies": {
+    "react": "^15.1.0",
```
Comment (leoasis) [inline package.json]: The changes you've made don't make the lib not work with previous versions of react, we should still support them if we can

Comment (vmasto) [inline package.json]: All right, I'll revert to `"react": ">=0.13.3 <16.0.0"`.

Comment (vmasto) [inline package.json]: Done, I kept `react-dom` as a devDependency just to be able for us to modernize the examples.
(plus there's no other way for us to make them work anyway apart from hardcoding React's version to pre-reactdom)

Comment (leoasis) [inline package.json]: Yeah that's fine as long as it's a dev dependency

**Comment by leoasis** [comment]
Thanks @vmasto for the contribution!

**Pull request merged**